rapids/README.md

```diff
@@ -14,21 +14,113 @@ action supports Dask and Spark runtimes for RAPIDS on
 [best practices](/README.md#how-initialization-actions-are-used) of using
 initialization actions in production.
 
-## Spark RAPIDS XGBoost
+## Spark RAPIDS Accelerator
+
+### Prerequisites
+
+To use Spark Rapids SQL plugin, XGBoost4j with Spark 3
+
+*   Apache Spark 3.0.0 RC1+
+*   Hardware Requirements
+    *   NVIDIA Pascal™ GPU architecture or better (V100, P100, T4 and later)
+    *   Multi-node clusters with homogenous GPU configuration
+*   Software Requirements
+    *   NVIDIA driver 440.33+
+    *   CUDA v10.2/v10.1/v10.0
+    *   NCCL 2.4.7 and later
 
 This section describes how to create
-[Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster with Spark
-RAPIDS XGBoost.
+[Google Cloud Dataproc](https://cloud.google.com/dataproc) cluster with
+[Spark RAPIDS SQL plugin](https://github.com/NVIDIA/spark-rapids) and
+[XGBoost4j](https://github.com/rapidsai/spark-examples/tree/support-spark3.0).
+
+### Step 1. Create Dataproc cluster with Spark RAPIDS Accelerator
+
+The following command will create a new Dataproc cluster named `CLUSTER_NAME`
+with installed GPU drivers, Spark RAPIDS Accelerator, Spark RAPIDS XGBoost
+libraries and Jupyter Notebook.
+
+A few notes:
+
+*   `spark.yarn.unmanagedAM.enabled=false` is set since it current breaks
+    SparkUI, will remove later
+*   for better GPU performance it's recommended to remove IO bottleneck as much
+    as possible, that includes faster disk/networking.
+*   Adjust spark properties in cluster creation command to the hardware
+    availability
+*   For best practice, please refer to NVIDIA
+    [getting started guide](https://nvidia.github.io/spark-rapids/)
+
+```bash
+export CLUSTER_NAME=<cluster_name>
+export GCS_BUCKET=<your bucket for the logs and notebooks>
+export REGION=<region>
+export NUM_GPUS=1
+export NUM_WORKERS=2
+
+gcloud beta dataproc clusters create $CLUSTER_NAME  \
```

Comment:
We can remove `beta` here as `component gateway` is now GA.

Reply:
done